encrypted-media-respec.html

```diff
@@ -2990,23 +2990,11 @@ <h2>
               </p>
             </div>
             <div>
-              <pre class="idl">
-                  enum HDCPVersion {
-                      "1.0",
-                      "1.1",
-                      "1.2",
-                      "1.3",
-                      "1.4",
-                      "2.0",
-                      "2.1",
-                      "2.2",
-                      "2.3",
-                  };
-                </pre>
               <p>
-                The HDCP Policy is represented by {{MediaKeysPolicy/minHdcpVersion}}. When set, the
-                policy requirement will be fulfilled if the system supports
-                {{MediaKeysPolicy/minHdcpVersion}} on the current display.
+                The HDCP Policy is represented by {{MediaKeysPolicy/minHdcpVersion}}. If the system
+                can enable the HDCP version specified or higher, then the policy will result in a
+                {{MediaKeyStatus}} of {{MediaKeyStatus/"usable"}}. Implementations MUST support
+                {{MediaKeysPolicy/minHdcpVersion}} values defined in [[EME-HDCP-VERSION-REGISTRY]].
```

Comment:
Apologies for the late review.

I'm not sure what that requirement means. A registry evolves over time. If "2.4" gets added to the registry, it seems fine for an implementation not to support the *value* initially, the algorithm will just throw a `NotSupportedError` error in that case.

Also, registries are informative in essence, so a normative reference to them is strange. I would replace with a non-normative "The [[?EME-HDCP-VERSION-REGISTRY]] provides the mapping from {{MediaKeysPolicy/minHdcpVersion}} values to HDCP specifications".

Reply:
Not late at all, happy to get feedback anytime.

I'm happy to make this change, but I was just borrowing the language from elsewhere in the spec, "Implementations MUST support the common formats defined in [[EME-INITDATA-REGISTRY]]..."

Reply:
I just found the relevant part of the Process is https://www.w3.org/2023/Process-20231103/#reg-ref-specifications

Reply:
https://github.com/w3c/encrypted-media/pull/540

Reply:
> I'm happy to make this change, but I was just borrowing the language from elsewhere in the spec, "Implementations MUST support the common formats defined in [[EME-INITDATA-REGISTRY]]..."

Yes, I would say that this needs fixing as well ;) The formats that must be supported need to appear explicitly in the spec itself, as illustrated in the example @chrisn referenced from the Process.

Reply:
OK, so the key question is: are we comfortable with these being non-normative? If not, then adding a registry isn't the right direction. Sorry, my fault, should have gone through this earlier.